tests/trap.etest

```diff
@@ -7,6 +7,55 @@
 # as published by the Apache Software Foundation, either version 2 of the License, or (at your option) any later
 # version.
 
+# Basic trap_add and trap_get sanity test with 1 signal.
+ETEST_trap_add()
```

Comment:
And also tests! Great

Reply:
:-). And in true TDD fashion, these tests were failing without my fix. That's always a good place to start before you fix the bug :-)